src/stylesheets/_variables.scss

```diff
@@ -21,6 +21,9 @@ $font-sizes: (
   sm: 0.8125rem,
   md: 0.9375rem,
   lg: 1.0625rem,
+  xl: 1.75rem,
+  xxl: 2.125rem,
+  sxl: 4rem
```

Comment:
@rafayet-monon Does`sxl` mean `super xl`? 😛
For this case, we can make it as `xxxl`, `xxxxl` and so on.